NvimView/NvimView/com.qvacua.NvimView.vim

```diff
@@ -1,2 +1,7 @@
 set mouse=a
 set title
+
+autocmd VimEnter,ColorScheme * :highlight default VimrDefaultCursor gui=reverse guibg=NONE guifg=NONE
+set guicursor=a:block-VimrDefaultCursor
+autocmd VimEnter,ColorScheme * :highlight default VimrInsertCursor guibg=fg
+set guicursor=i:ver25-VimrInsertCursor
```

Comment:
I don't really understand what's going on here, but it seems to work 😀

Reply:
This creates two "highlighting groups", "VimrDefaultCursor" and "VimrInsertCursor" (I hope those do not clash with anyone's custom vimrc). It then defines two cursor shapes, one for INSERT mode and one for all other modes, to use the newly created highlighting definitions.
Because custom highlighting groups are cleared on a colorscheme change, the ":highlight …" command is added as an autocmd, so the highlighting groups are recreated after every colorscheme change.